apps/client/src/components/elements/Button/button.module.scss

```diff
@@ -17,19 +17,22 @@
     border-radius: 30px;
   }
 
-  &:hover {
-  }
+  // &:hover {
+  // }
 
-  &:focus,
-  &:focus-visible {
-  }
+  // &:focus,
+  // &:focus-visible {
+  // }
```

Comment:
either remove it or leave it, applies to others.

I had left it intentionally so I can add the button different states